main.go

```diff
@@ -526,9 +529,7 @@ func startHealthProxy(ctx context.Context, wg *sync.WaitGroup, addresses ...stri
 				defer resp.Body.Close()
 
 				if resp.StatusCode != http.StatusOK {
-					body := []byte{}
-
-					_, err = resp.Body.Read(body)
+					body, err := ioutil.ReadAll(resp.Body)
```

Comment:
Can this use `io.ReadAll` instead? https://pkg.go.dev/io/ioutil#ReadAll is deprecated